library/groovy

```diff
@@ -1,233 +1,136 @@
 Maintainers: Keegan Witt <keeganwitt@gmail.com> (@keeganwitt)
 GitRepo: https://github.com/groovy/docker-groovy.git
 
-Tags: 2.4.15-jdk7, 2.4-jdk7, jdk7
+Tags: 2.5.0-jdk7, 2.5-jdk7, jdk7
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
-GitCommit: 420f3af36206e1e6ddb1b715bfcc9f0591c8501f
+GitCommit: 2406091b41b2ca547e9d5182ef5c6d53d3017ca3
 Directory: jdk7
 
-Tags: 2.4.15-jre7, 2.4-jre7, jre7
+Tags: 2.5.0-jre7, 2.5-jre7, jre7
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
-GitCommit: 420f3af36206e1e6ddb1b715bfcc9f0591c8501f
+GitCommit: 2406091b41b2ca547e9d5182ef5c6d53d3017ca3
 Directory: jre7
 
-Tags: 2.4.15-jdk7-alpine, 2.4-jdk7-alpine, jdk7-alpine
+Tags: 2.5.0-jdk7-alpine, 2.5-jdk7-alpine, jdk7-alpine
 Architectures: amd64, arm32v6, arm64v8, i386, ppc64le, s390x
-GitCommit: 420f3af36206e1e6ddb1b715bfcc9f0591c8501f
+GitCommit: 2406091b41b2ca547e9d5182ef5c6d53d3017ca3
 Directory: jdk7-alpine
 
-Tags: 2.4.15-jre7-alpine, 2.4-jre7-alpine, jre7-alpine
+Tags: 2.5.0-jre7-alpine, 2.5-jre7-alpine, jre7-alpine
 Architectures: amd64, arm32v6, arm64v8, i386, ppc64le, s390x
-GitCommit: 420f3af36206e1e6ddb1b715bfcc9f0591c8501f
+GitCommit: 2406091b41b2ca547e9d5182ef5c6d53d3017ca3
 Directory: jre7-alpine
 
-Tags: 2.4.15-jdk8, 2.4-jdk8, jdk8, 2.4.15-jdk, 2.4-jdk, jdk
+Tags: 2.5.0-jdk8, 2.5-jdk8, 2.5.0-jdk, 2.5-jdk, jdk
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
-GitCommit: 420f3af36206e1e6ddb1b715bfcc9f0591c8501f
+GitCommit: 2406091b41b2ca547e9d5182ef5c6d53d3017ca3
 Directory: jdk8
 
-Tags: 2.4.15-jre8, 2.4-jre8, jre8, 2.4.15-jre, 2.4-jre, jre, 2.4.15, 2.4
+Tags: 2.5.0-jre8, 2.5-jre8, 2.5.0-jre, 2.5-jre, 2.5.0, 2.5, jre, latest
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
-GitCommit: 420f3af36206e1e6ddb1b715bfcc9f0591c8501f
+GitCommit: 2406091b41b2ca547e9d5182ef5c6d53d3017ca3
 Directory: jre8
 
-Tags: 2.4.15-jdk8-alpine, 2.4-jdk8-alpine, jdk8-alpine, 2.4.15-jdk-alpine, 2.4-jdk-alpine, jdk-alpine
+Tags: 2.5.0-jdk8-alpine, 2.5-jdk8-alpine, 2.5.0-jdk-alpine, 2.5-jdk-alpine, jdk-alpine
 Architectures: amd64, arm32v6, arm64v8, i386, ppc64le, s390x
-GitCommit: 420f3af36206e1e6ddb1b715bfcc9f0591c8501f
+GitCommit: 2406091b41b2ca547e9d5182ef5c6d53d3017ca3
 Directory: jdk8-alpine
 
-Tags: 2.4.15-jre8-alpine, 2.4-jre8-alpine, jre8-alpine, 2.4.15-jre-alpine, 2.4-jre-alpine, jre-alpine, 2.4.15-alpine, 2.4-alpine
```

Comment:
Did you mean to drop these tags from here and above? `jdk8` `jre8` `jdk8-alpine` `jre8-alpine`

Reply:
No, I didn't.  Thank you for catching this.